cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/paymentOptions/paymentOptionsSummary.isml

```diff
@@ -3,7 +3,7 @@
         <isif condition="${payment.paymentMethod === 'CREDIT_CARD'}">
             <isinclude template="checkout/billing/paymentOptions/creditCardSummary"/>
         </isif>
-        <isif condition="${payment.paymentMethod === 'Adyen'}">
+        <isif condition="${payment.paymentMethod === 'Adyen' || payment.paymentMethod == 'AdyenPOS'}">
```

Comment:
If possible, It would be nice to use constants for Adyen and AdyenPOS.

Reply:
As discussed, we will use a constant file for backend constants for now